app/views/posts/index.html.erb

```diff
@@ -1 +1,8 @@
-<h1> this is a list of posts of a given user </h1>
\ No newline at end of file
+<% @posts.each do |post|%>
+    <div class="post">
+        <h3><%= link_to post.title, {:action=> 'show', :id => post.id} %></h3>
+        <p><%= post.text %></p>
+        <p>Comments: <%= post.comments.count %></p>
+        <p>Likes: <%= post.likes.count %></p>
+    </div>
+<%end%>
```

Comment:
- [ ] When we render a post, we should also render the five most recent comments of that post. That is suggested by this section of the wireframe 👇🏼:

  ![image](https://user-images.githubusercontent.com/45002654/167712269-068710cf-ead5-470a-9537-f86c71d4b836.png)
  
  Please add code that will display the five most recent comments in this view.